### PR TITLE
Casmpet 5680

### PR DESCRIPTION
**Pull request by dzou-hpe**
## Summary and Scope

add images for argo workflow

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves CASMPET-5680
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+
+name: quay.io/argoproj/workflow-controlle:v3.3.6
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:26]:
```suggestion
name: quay.io/argoproj/workflow-controller:v3.3.6
```

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/quay.io.argoproj.workflow-controlle.v3.3.6.yaml
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:33]:
```suggestion
      - .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml
```

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      - quay.io.argoproj.workflow-controlle.v3.3.6/**
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:34]:
```suggestion
      - quay.io.argoproj.workflow-controller.v3.3.6/**
```

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/quay.io.argoproj.workflow-controlle.v3.3.6.yaml
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:39]:
```suggestion
      - .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml
```

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      - quay.io.argoproj.workflow-controlle.v3.3.6/**
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:40]:
```suggestion
      - quay.io.argoproj.workflow-controller.v3.3.6/**
```

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    env:
+      CONTEXT_PATH: quay.io/argoproj/workflow-controlle/v3.3.6
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:50]:
```suggestion
      CONTEXT_PATH: quay.io/argoproj/workflow-controller/v3.3.6
```

File: .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+      DOCKER_REPO: artifactory.algol60.net/csm-docker/stable/quay.io/argoproj/workflow-controlle
```
Comment (bo-quan) [inline .github/workflows/quay.io.argoproj.workflow-controller.v3.3.6.yaml:51]:
```suggestion
      DOCKER_REPO: artifactory.algol60.net/csm-docker/stable/quay.io/argoproj/workflow-controller
```

**Review by bklei — APPROVED** [review]

File: quay.io/argoproj/argoexec/v3.3.6/Dockerfile [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+FROM quay.io/argoproj/argoexec:v3.3.6
```
Comment (bo-quan) [inline quay.io/argoproj/argoexec/v3.3.6/Dockerfile:24]: add a newline at the end

**Review by bo-quan — CHANGES_REQUESTED** [review]

**Review by bo-quan — APPROVED** [review]

**Pull request merged**